docs/user-guide/work-packages/set-change-dates/README.md

```diff
@@ -0,0 +1,138 @@
+---
+sidebar_navigation:
+  title: Set and change dates
+  priority: 890
+description: How to use the date picker to set and update start and finish of work packages
+keywords: date picker start finish dates change modify update relations work package
+---
+
+# Set and change start and finish dates of work packages
+
+| Feature                                                      | Documentation for                                         |
+| ------------------------------------------------------------ | --------------------------------------------------------- |
+| [Set start and finish dates](#set-start-and-finish-dates)    | How to set start and finish dates, and how to change them |
+| [Scheduling mode](#scheduling-mode)                          | Enabling manual scheduling and what it does               |
+| [Warning and information banners](#warning-and-information-banner) | What the different warning and information banners mean   |
+
+## Set start and finish dates
+
+You can change the start and finish dates of work packages opening the date picker. 
+
+![standard-date-picker](standard-date-picker.png)
+
+1. [Manual scheduling](#scheduling-mode) checkbox
+2. Start and finish date input fields
+3. Mini calendars for date range preview
+4. Save and cancel buttons
+
+You can open the date picker for a work package from a number of different places:
+
+- By clicking on the date field in the work package [details view](../../work-packages/work-package-views/#work-package-full-screen-view)
```

Comment:
This path is technically working, but just FYI: 
it is navigating two levels up `../../` just to enter `work-packages` again.
Since we are already in sub-folder of `work-packages` It can be shortened to
`[details view](../work-package-views/#work-package-full-screen-view)`